calico-cloud/reference/resources/felixconfig.mdx

```diff
@@ -186,6 +186,7 @@ spec:
 | mtuIfacePattern                       | Pattern used to discover the host's interface for MTU auto-detection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | regex                                                              | string                                  | `^((en                                                                                                                                                                                                                                                                                                                                                                                                                                            | wl     | ww       | sl  | ib)[opsx].\* | (eth | wlan | wwan).\*)` |
+| bpfForceTrackPacketsFromIfaces        | BPFForceTrackPacketsFromIfaces in BPF mode, forces traffic from these interfaces to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be tracked by Linux conntrack.  Should only be used for interfaces that are not used for the Calico fabric.  For example, a docker bridge device for non-Calico-networked containers.                                                                                                                                                                                                                                                                                                                                                                                                  | A list of strings                                                  | A list of strings                       | docker+                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
```suggestion
| bpfForceTrackPacketsFromIfaces        | Forces traffic from these interfaces in BPF mode to skip Calico's iptables NOTRACK rule, allowing traffic from those interfaces to be tracked by Linux conntrack.  Use only for interfaces that are not used for the Calico fabric, for example, a docker bridge device for non-Calico-networked containers.                                                                                                                                                                                                                                                                                                                                                                                                  | A list of strings                                                  | A list of strings                       | docker+                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

Reply:
Update description text as per suggested change
NB: updated all 3x instances for consistency